### PR TITLE
AboutUs Images

### PR DESCRIPTION
**Pull request by ItzAbraham**
# *AboutUs Images*

## Description (*What does this PR do?*)

It's a correction of the Names used to save the Teams Images and also the correction of the Frontend Team Arrangement

## Related Issue(s)

- [ ] Issue 1
- [ ] Issue 2

## Checklist

- [x] I have tested the changes locally and they work as expected. Yes
- [x] The code follows the project's coding standards and conventions. Yes
- [ ] I have added appropriate comments and documentation where necessary.
- [x] All tests passed successfully. Yes
- [ ] I have updated the necessary dependencies.
- [ ] I ran ``` npm run build ``` and it built without errors

## Changes Made

Correction of the Image Names 
Arrangement of the Frontend Team

## Screenshots

Ensure you add screenshots or GIFs to showcase the changes visually.
Screenshot (40)

## Additional Notes

Add any additional notes or context that may be helpful for the reviewer.

### CODE REVIEW

File: .hintrc
Comment (Christiano112) [inline .hintrc:1]: What's this file for?

File: src/data/teamMembers/backendTeam.js [hunk 1]
```diff
@@ -1,27 +1,32 @@
 export const teamBackend = [
     {
-        name: "Aslam Dhikrullahi",
-        title: "Design Lead",
-        image: "/test_img1.jpg",
+        name: "Oluwafikayomi Soetan",
+        title: "Backend Development Lead",
+        image: "/Oluwafikayomi.png",
     },
```
Comment (Christiano112) [inline src/data/teamMembers/backendTeam.js:6]: The name of the images should be in this format:
toyin_BE
precious_BE
abraham_FE
miracle_PD

etc

**Review by Christiano112 — CHANGES_REQUESTED** [review]

**Comment by Christiano112** [comment]
Kindly rename the images

**Comment by Christiano112** [comment]
Kindly pull from develop and see to the object id updates

**Comment by Christiano112** [comment]
why have you not fixed this conflicts

File: src/data/teamMembers/backendTeam.js (hunk 1 above)
Comment (ItzAbraham) [reply src/data/teamMembers/backendTeam.js:6]: okay sir

**Pull request closed without merging**